docs/manual/forms/examples/objects_prefill.rst

```diff
@@ -0,0 +1,165 @@
+#. Selecteer één van de (demo) DigiD inlogopties bij de formulierauthenticatiemethoden.
+#. Maak een gebruikersvariabele aan met de naam **Aanvraaggegevens** en datatype
+   ``object``.
+#. Klik het potlood-icoontje aan in de kolom "Prefill" van de gebruikersvariabele. Je
+   kan nu de opties instellen.
+
+    * Bij **Plugin** selecteer je "Objecten API". Er komen nu extra opties in beeld.
+    * Voor **API-groep** selecteer je de groep die aangemaakt is door een beheerder.
+    * Selecteer bij **Objecttype** het vergunning-objecttype wat eerder aangemaakt is.
+    * Selecteer bij **Versie** de meest recente versie.
+    * Bij **Path to auth attribute** kies je voor ``bsn``. De opties in deze lijst komen
+      uit het geselecteerde objecttype.
+
+    De instellingen moeten op de onderstaande screenshot lijken:
+
+    .. image:: _assets/product_prefill_options_1.png
+       :alt: Screenshot van de geselecteerde opties in de dropdowns.
+
+    Vervolgens stellen we in waar de individuele attributen toegekend worden.
+
+    * Scroll omlaag bij de instellingen totdat de "Variabelekoppleingen" in beeld staan.
+    * Klik op **Variabele toevoegen**, en kies voor de formuliervariabele
+      "Referentienummer", met brondpad ``identifier`` uit het objecttype.
+    * Klik op **Variabele toevoegen**, en kies voor de formuliervariabele
+      "Van", met brondpad ``startDate`` uit het objecttype.
+    * Klik op **Variabele toevoegen**, en kies voor de formuliervariabele
+      "Tot", met brondpad ``endDate`` uit het objecttype.
+    * Klik op **Variabele toevoegen**, en kies voor de formuliervariabele
+      "Opmerkingen / extra informatie", met brondpad ``remarks`` uit het objecttype.
+
+    .. image:: _assets/product_prefill_options_2.png
+       :alt: Screenshot met de variabelekoppelingen in een tabel.
+
+    * Klik op "Opslaan" om de instellingen te bewaren.
+
+#. Sla het formulier op.
+
+Formulier invullen
+==================
+
+Om de vooringevulde gegevens te zien moet je de gegevensreferentie meegeven in de
+formulierlink. Stel dat het formulier normaal beschikbaar is op
+``https://forms.example.com/voorbeeld/``, dan wordt de nieuwe URL met
+voorinvullen:
+
+.. code-block:: none
+
+    https://forms.example.com/voorbeeld/?initial_data_reference=05a0972b-c604-4b58-bd7d-6d456fb2987b
+
+Hier gebruik je het UUID van het Object dat de brongegevens bevat.
+
+.. todo:: Fix https://github.com/open-formulieren/open-forms/issues/4918 as these URLs
+   aren't currently working as intended.
+
+Bonus: registreren in de Objecten API
+=====================================
+
+Je kan dit formulier ook weer registreren in de Objecten API. Belangrijk hierbij is dat
+je dan:
+
+* Bij de variabelen koppel je de ``auth_bsn`` statische variabele aan het
+  ``bsn``-attribuut in het objecttype.
+* Bij de variabelen koppel je de registratievariabele ``public_reference`` aan het
+  ``identifier``-attribuut in het objecttype.
```

Comment:
To make the sentences flow better with the `Belangrijk hierbij is dat je dan` part
```suggestion
* Bij de variabelen de ``auth_bsn`` statische variabele aan het
  ``bsn``-attribuut in het objecttype koppelt.
* Bij de variabelen de registratievariabele ``public_reference`` aan het
  ``identifier``-attribuut in het objecttype koppelt.
```